app/package.json

```diff
@@ -32,6 +32,7 @@
 		"algoliasearch": "^3.26.0",
 		"axios": "^0.18.0",
 		"babel-plugin-inline-dotenv": "^1.1.2",
+		"bull": "^3.4.1",
```

Comment:
can probably pull this - shouldn't be connecting to `bull` from the frontend app

Reply:
Thought this was pulled from package.json. Removing now.